### DISCUSSION TITLE
HEIC Decoder

### DISCUSSION THREAD

**Discussion by pradeep-patil2024** [SixLabors/ImageSharp#2772] [Ideas]
Hi Team, 

Need HEIC decoder to convert image to supported encoder formats.

**Comment by JimBobSquarePants** [comment]
See https://github.com/SixLabors/ImageSharp/pull/2633

Please be aware that when opening a discussion, it's considered poor behavior to simply ask for a feature with no detail regarding requirements. These libraries are developed in the author's spare time.
